src/main/kotlin/ink/abb/pogo/scraper/Settings.kt

```diff
@@ -93,6 +94,17 @@ class Settings(val properties: Properties) {
         listOf()
     }
 
+    val autoEvolve = getPropertyIfSet("list of pokemon you want to evolve when able to", "auto_evolve", "CATERPIE,PIDGEY,WEEDLE", String::toString).split(",")
+
+    //This method only exists because I wasn't sure if this data was stored somewhere else. If it is then this can be removed.
+    private fun getCandyByPokemon(): Map<Int, Int> {
+            val lines = File("pokemon-candy.csv").readLines()
```

Comment:
That's a good idea, I just ran this and it evolved 87 Pidgies in less than a minute (build in delays?).  Perhaps best to leave lucky egg usage on manual, they're rare to come by without spending.